pandas/_libs/src/vendored/ujson/python/ujson.c

```diff
@@ -56,9 +56,11 @@ PyObject *JSONToObj(PyObject *self, PyObject *args, PyObject *kwargs);
   "encode_html_chars=True to encode < > & as unicode escape sequences."
 
 static PyMethodDef ujsonMethods[] = {
-    {"ujson_dumps", (PyCFunction)objToJSON, METH_VARARGS | METH_KEYWORDS,
+    {"ujson_dumps", (PyCFunction)(void (*)(void))objToJSON,
```

Comment:
I find this cast here a bit strange but it is what CPython has in their documentation for functions that accept kwargs

https://docs.python.org/3/extending/extending.html#keyword-parameters-for-extension-functions